Plugins/Util/Util.cpp

```diff
@@ -100,5 +104,30 @@ ArgumentStack Util::Hash(ArgumentStack&& args)
     return stack;
 }
 
+ArgumentStack Util::GetCustomToken(ArgumentStack&& args)
+{
+    ArgumentStack stack;
+    std::string retVal = "";
+    auto *pTlk = API::Globals::TlkTable();
+    auto *pTokens = pTlk->m_pTokensCustom;
+    int numTokens = pTlk->m_nTokensCustom;
+
+    const auto tokenNumber = Services::Events::ExtractArgument<int32_t>(args);
+
+    CTlkTableTokenCustom token;
+    token.m_nNumber = tokenNumber < 0 ? 0 : tokenNumber;
```

Comment:
There's no reason to set this. `token` here is just used to store the result.

Reply:
If `tokenNumber` is negative, just return "" immediately.

Reply:
Done